src/main/java/Collections/Book.java

```diff
@@ -0,0 +1,42 @@
+package Collections;
+
+import java.util.Objects;
+
+public class Book {
+    private final String author;
+    private final String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public String getAuthor() {
+        return author;
+    }
+
+    public Book(String author, String name) {
+        this.author = author;
+        this.name = name;
+    }
+
+    @Override
+    public String toString() {
+        return "Book{" +
+                "author='" + author + '\'' +
+                ", name='" + name + '\'' +
+                '}';
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Book book = (Book) o;
+        return Objects.equals(author, book.author) && Objects.equals(name, book.name);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(author, name);
+    }
```

Comment:
Observe the structure of the class. Constructors come first, then getters and setters, then hashCode and equals, then toString.